src/cpp/rtps/builtin/BuiltinProtocols.cpp

```diff
@@ -173,6 +176,8 @@ void BuiltinProtocols::transform_server_remote_locators(
 {
     // TODO(jlbueno) The access to the list should be protected with the PDP mutex but requires a refactor on PDPClient
     // and PDPServer: read the remote servers list after the initialization of PDP.
```

Comment:
Is this comment still valid? Now the access to the list is protected, but may be the refactor is still needed?

Reply:
íbidem